Scripts/ci-deploy-snapshot.sh

```diff
@@ -3,19 +3,12 @@
 set -e
 
 #
-# Nuget repository
+# NuGet repository
 #
 DEFAULT_BONITOO_NUGET_URL="https://apitea.com/nexus/service/local/nuget/bonitoo-nuget/"
 BONITOO_NUGET_URL="${BONITOO_NUGET_URL:-$DEFAULT_BONITOO_NUGET_URL}"
 
 #
-# Deploy to Preview repository
+# Deploy to Preview NuGet repository
 #
-dotnet pack Client.Core --version-suffix=dev.$CIRCLE_BUILD_NUM
-dotnet pack Client.Legacy --version-suffix=dev.$CIRCLE_BUILD_NUM
-dotnet pack Client --version-suffix=dev.$CIRCLE_BUILD_NUM
-dotnet pack Client.Linq --version-suffix=dev.$CIRCLE_BUILD_NUM
-dotnet nuget push ./Client.Core/bin/Debug/InfluxDB.Client.Core.*-dev.$CIRCLE_BUILD_NUM.nupkg -s ${BONITOO_NUGET_URL} -k ${BONITOO_SNAPSHOT_APIKEY} -sk ${BONITOO_SNAPSHOT_APIKEY}
-dotnet nuget push ./Client.Legacy/bin/Debug/InfluxDB.Client.Flux.*-dev.$CIRCLE_BUILD_NUM.nupkg -s ${BONITOO_NUGET_URL} -k ${BONITOO_SNAPSHOT_APIKEY} -sk ${BONITOO_SNAPSHOT_APIKEY}
-dotnet nuget push ./Client/bin/Debug/InfluxDB.Client.*-dev.$CIRCLE_BUILD_NUM.nupkg -s ${BONITOO_NUGET_URL} -k ${BONITOO_SNAPSHOT_APIKEY} -sk ${BONITOO_SNAPSHOT_APIKEY}
-dotnet nuget push ./Client.Linq/bin/Debug/InfluxDB.Client.Linq.*-dev.$CIRCLE_BUILD_NUM.nupkg -s ${BONITOO_NUGET_URL} -k ${BONITOO_SNAPSHOT_APIKEY} -sk ${BONITOO_SNAPSHOT_APIKEY}
\ No newline at end of file
+dotnet nuget push ./NuGetPackages/InfluxDB.Client.*.nupkg -s ${BONITOO_NUGET_URL} -k ${BONITOO_SNAPSHOT_APIKEY} -sk ${BONITOO_SNAPSHOT_APIKEY}
```

Comment:
lease ensure to include dev.$CIRCLE_BUILD_NUM. as part of the regex. This is crucial to guarantee that only the packages generated in the current CI run are pushed.
```suggestion
dotnet nuget push ./NuGetPackages/InfluxDB.Client.*-dev.$CIRCLE_BUILD_NUM.nupkg -s ${BONITOO_NUGET_URL} -k ${BONITOO_SNAPSHOT_APIKEY} -sk ${BONITOO_SNAPSHOT_APIKEY}
```